templates/frontOffice/default/product.html

```diff
@@ -230,11 +230,16 @@ <h1 class="name"><span itemprop="name" content="{$TITLE}">{$TITLE}</span><span i
                     {form_field field="append"}
                         <input type="hidden" name="{$name}" value="1">
                     {/form_field}
+
                     {form_field form=$form field="success_url"}
                         <input type="hidden" name="{$name}" value="{url path='/cart'}">
                     {/form_field}
 
-                    {if $form_error}<div class="alert alert-error">{$form_error_message}</div>{/if}
+                    {form_field form=$form field="error_url"}
```

Comment:
Same here, I'm not sure this change need to be in this PR

Reply:
Original product.html has been restored